<launch>
    <!-- Caricamento parametri nbv -->
    <rosparam file="$(find nbv_planner)/params/nbv_params.yaml" command="load"/>
    
    <!-- Carico confini workspace -->
    <param name="/workspace/x_min" value="-1.0"/>
    <param name="/workspace/x_max" value="14.0"/>
    <param name="/workspace/y_min" value="-1.0"/>
    <param name="/workspace/y_max" value="14.0"/>

    <!-- Zeno + Lidar -->
    <include file="$(find models)/launch/my_zeno.launch"/>

    <!-- Nodo di gmapping -->
    <include file="$(find models)/launch/mapping.launch"/>

    <!-- NBV planner -->
    <node pkg="nbv_planner" type="nbv_planner.py" name="nbv_planner" output="screen" />

    <!-- Visualizzazione workspace -->
    <node pkg="common_fun" type="workspace_pub.py" name="workspace_pub" output="screen" />

    <!-- Rviz -->
    <node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find models)/config/unicycle.rviz"/>
    
</launch>